scripts/ensure-flow.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -eu
+comm -13 -z \
+  <(git grep -z --name-only -e @flow -e @no-flow | sort -z) \
+  <(find src config scripts -name '*.js' -print0 | sort -z) \
+  | tr '\0' '\n' \
+  | diff -au /dev/null -
```

Comment:
We don't need the `-a` flag to `diff`, because of the `tr` on the previous line.

Reply:
Fixed.